docs/components/motor/gpio.md

```diff
@@ -21,6 +21,7 @@ Find more information in the [encoded motor documentation](/components/motor/enc
 {{< /alert >}}
 
 To configure a DC motor as a component of your machine, first configure the [board](/components/board/) to which the motor driver is wired.
+Make sure to physically connect your motor to your board and turn it on.
```

Comment:
This doesn't make sense since the first sentence implies the motor driver is already wired to the board.
```suggestion
To configure a DC motor as a component of your machine, first make sure the motor is wired to a suitable motor driver, which is in turn wired to a board.
Configure the [board](/components/board/) to which the motor driver is wired.
```

Reply:
@sguequierre This comment applies to byj motor too